portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,216 @@
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <hr class="m-0" />
+            <!-- Photos-->
+            <section class="resume-section" id="photos">
+                <div class="resume-section-content">
+                    <h2 class="mb-5">Photos I've taken</h2>
+                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
+                        <div class="flex-grow-1">
+                    <!-- Carousel -->
+                    <div id="demo" class="carousel slide" data-bs-ride="carousel">
+
+                        <!-- Indicators/dots -->
+                        <div class="carousel-indicators">
+                        <button type="button" data-bs-target="#demo" data-bs-slide-to="0" class="active"></button>
+                        <button type="button" data-bs-target="#demo" data-bs-slide-to="1"></button>
+                        <button type="button" data-bs-target="#demo" data-bs-slide-to="2"></button>
+                        </div>
+                        
+                        <!-- The slideshow/carousel -->
+                        <div class="carousel-inner">
+                        <div class="carousel-item active">
+                            <img src="/images/statue.jpg" alt="Armenia" class="d-block" style="width:100%">
+                        </div>
+                        <div class="carousel-item">
+                            <img src="/images/Penang.jpg" alt="Penang" class="d-block" style="width:100%">
+                        </div>
+                        <div class="carousel-item">
+                            <img src="/images/KL.jpg" alt="KL" class="d-block" style="width:100%">
+                        </div>
+                        </div>
+                        
+                        <!-- Left and right controls/icons -->
+                        <button class="carousel-control-prev" type="button" data-bs-target="#demo" data-bs-slide="prev">
+                        <span class="carousel-control-prev-icon"></span>
+                        </button>
+                        <button class="carousel-control-next" type="button" data-bs-target="#demo" data-bs-slide="next">
+                        <span class="carousel-control-next-icon"></span>
+                        </button>
+                    </div>
+                </div>
+                        <div class="flex-shrink-0"><span class="text-primary"></span></div>
+                    </div>
+                    <div class="d-flex flex-column flex-md-row justify-content-between">
+                        <div class="flex-grow-1">
+                            <div class="subheading mb-3">I  am not a huge fan of photography as I usually forget to take pictures and I love living in the moment. However, at the same time I really enjoy looking back at pictures and reliving through them.</div>
```

Comment:
Try to name your style classes something that either is specific to its purpose in the document or generic and applicable in most places. Ideally if we want to change the style of something in the html, it just requires a css and not an html change as well.

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming